core/src/org/javarosa/core/model/instance/TreeReference.java

```diff
@@ -221,9 +282,31 @@ public TreeReference clone() {
         return newRef;
     }
 
+    /**
+     * Return a copy of the TreeReference that doesn't include any of the
+     * TreeReferenceLevels. Useful when we are just going to overwrite the
+     * levels with new data anyways.
+     *
+     * @return a clone of this object that doesn't include any reference level
+     * data.
+     */
+    private TreeReference cloneWithoutLevels() {
```

Comment:
What does this do conceptually, exactly? only clones the instance name and root ref level? Might be a more explanatory name we can be using here.